gym/download_models.bash

```diff
@@ -0,0 +1,5 @@
+wget http://download.tensorflow.org/models/object_detection/tf2/20200711/ssd_mobilenet_v2_fpnlite_640x640_coco17_tpu-8.tar.gz
```

Comment:
I wonder if it makes sense to make it a bash function where you specify the URI and target directory to deploy the model.

Reply:
what you meant by that?

is not model deployment, it is a pretrained model, 
since .gitignore do not allow  to push pretrained model
I wrote this script to make it easy for people to download the pre trained model

Reply:
Not a big issue.  Just a style thing:
imagine if you could call a bash/cli just with 

"download_model 
 --uri http://download.tensorflow.org/models/object_detection/tf2/20200711/ssd_mobilenet_v2_fpnlite_640x640_coco17_tpu-8.tar.gz"

And behind the scenes would do all the unpacking and storing in the correct place.

Reply:
I see,
I've first considered to do what you are saying but I found it expensive \
devs are not supposed to know where the model will be downloaded from \
that's why I decided to hard code it that way \
I needed to hid the details as much as possible from devs \
so that the only thing they have to do is to use make